packages/graphql-server-module-graphiql/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "graphql-server-module-graphiql",
-  "version": "0.4.4",
+  "version": "0.6.1",
```

Comment:
Please, do not.
this should be managed by lerna when running `npm run release` ;)